include/xrpl/protocol/TxFormats.h

```diff
@@ -55,168 +55,25 @@ namespace ripple {
 // clang-format off
 enum TxType : std::uint16_t
 {
-    /** This transaction type executes a payment. */
-    ttPAYMENT = 0,
 
-    /** This transaction type creates an escrow object. */
-    ttESCROW_CREATE = 1,
-
-    /** This transaction type completes an existing escrow. */
-    ttESCROW_FINISH = 2,
-
-    /** This transaction type adjusts various account settings. */
-    ttACCOUNT_SET = 3,
-
-    /** This transaction type cancels an existing escrow. */
-    ttESCROW_CANCEL = 4,
-
-    /** This transaction type sets or clears an account's "regular key". */
-    ttREGULAR_KEY_SET = 5,
+#pragma push_macro("TRANSACTION")
+#undef TRANSACTION
+#define TRANSACTION(tag, value, name, fields) tag = value,
+#include <xrpl/protocol/transactions.h>
+#undef TRANSACTION
+#pragma pop_macro("TRANSACTION")
```

Comment:
For consistency, please follow the same style as in Feature.h (vertical spacing)